### ISSUE TITLE
bug: core card: card avatar missing text alternative for screen reader

### ISSUE THREAD

**Issue by Seamoo13** [SAP/fundamental-ngx#3545]
#### Is this a bug, enhancement, or feature request?

bug

#### Briefly describe your proposal.

Currently there is no text alternative for the avatar in the card header. A text alternative should be provided for the screen reader (TTS) and images should have a tooltip
env: https://fundamental-ngx.netlify.app/#/core/card

#### If this is a bug, please provide steps for reproducing it.

1. Enable a screen reader (TTS)
2. navigate to a card with an avatar
3. click on the header section and pay attention to the screen reader output. (notice there is no mention of an avatar/image)
4. click on the avatar to elicit it specifically (only navigation category is found)

![2020-10-05_2](https://user-images.githubusercontent.com/47522152/95052356-75531500-06f7-11eb-9162-00c095b2db34.png)
